docs/tutorials/scenario-simulation/installation.md

```diff
@@ -0,0 +1,58 @@
+# Installation
+
+This document contains step-by-step instruction on how to build [AWF Autoware Core/Universe](https://github.com/autowarefoundation/autoware) with `scenario_simulator_v2`.
+
+## Prerequisites
+
+1. Ubuntu 20.04 machine
+2. Hardware with CUDA 11.1 capable graphics card
+3. ROS2 Galactic Geochelone desktop version [installed](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html) and sourced:
```

Comment:
I think it's better to write these requirements in https://autowarefoundation.github.io/autoware-documentation/main/installation and refer to the page.

Related: https://github.com/autowarefoundation/autoware/discussions/281

@LalithVipulananthan What do you think about it?

Reply:
I agree. Something like this could be written here instead:

> ## Prerequisites 
>
> 1. [A machine that supports the minimum requirements to run Autoware](../installation)
> 2. [Autoware has been built and installed](../installation/#installing-autoware)